jdbc/src/main/java/org/apache/metamodel/jdbc/JdbcDataContext.java

```diff
@@ -99,6 +99,9 @@
     public static final String DATABASE_PRODUCT_HIVE = "Apache Hive";
     public static final String DATABASE_PRODUCT_SQLITE = "SQLite";
     public static final String DATABASE_PRODUCT_IMPALA = "Impala";
+		
+    private static final String DEFAULT_SCHEMA_NAME_SQLSERVER = "dbo";
+    private static final String RESULT_SET_COLUMN_NAME_SCHEMA_SQLSERVER = "TABLE_SCHEM";
```

Comment:
I would propose a different name for this constant, because `'TABLE_SCHEM"` isn't SQL Server specific. I would instead call it TABLE_SCHEMA_NAME_IDENTIFIER or SCHEMA_NAME_IDENTIFIER or something similar.

Reply:
Sure